packages/cli/tsup.config.ts

```diff
@@ -21,7 +21,7 @@ export default defineConfig({
   entry: ["src/index.ts", "src/mud.ts"],
   target: "esnext",
   format: ["esm"],
-  dts: false,
```

Comment:
some of these were `false` and unclear why, so I enabled for now to see if CI passes